loader/loader.go

```diff
@@ -169,6 +169,7 @@ func Load(config *compileopts.Config, inputPkgs []string, clangHeaders string, t
 			}
 			return nil, err
 		}
+
```

Comment:
Can you remove this empty line? It is causing a conflict.

I have changed this PR to use the dev branch, see: https://github.com/tinygo-org/tinygo/blob/release/CONTRIBUTING.md#how-to-use-our-github-repository

Reply:
sorry I branched off from release branch and that is why this conflict happens. Thank you for the pointer, and keep that in mind